packages/flutter/lib/src/material/tabs.dart

```diff
@@ -269,7 +269,10 @@ class _TabStyle extends AnimatedWidget {
   final TabBarThemeData defaults;
   final Widget child;
 
-  MaterialStateColor _resolveWithLabelColor(BuildContext context) {
+  MaterialStateColor _resolveWithLabelColor({
+    required BuildContext context,
+    IconThemeData? iconTheme,
+  }) {
```

Comment:
nit: could we make the build context a positional arg?

```suggestion
  MaterialStateColor _resolveWithLabelColor(
    BuildContext context, {
    IconThemeData? iconTheme,
  }) {
```

It would reduce verbosity and would be consistent with methods like [InheritedModel.inheritFrom](https://main-api.flutter.dev/flutter/widgets/InheritedModel/inheritFrom.html) and [Navigator.of](https://main-api.flutter.dev/flutter/widgets/Navigator/of.html).
Not a strong preference though :)